packages/mongodb-chatbot-server/src/routes/conversations/createResponse.test.ts

```diff
@@ -0,0 +1,561 @@
+import "dotenv/config";
+import request from "supertest";
+import { Express } from "express";
+import { DEFAULT_API_PREFIX } from "../../app";
+import { makeTestApp } from "../../test/testHelpers";
+
+jest.setTimeout(100000);
+
+describe("POST /conversations/completion", () => {
+  const endpointUrl = `${DEFAULT_API_PREFIX}/conversations/completion`;
+  let app: Express;
+  let ipAddress: string;
+  let origin: string;
+
+  beforeEach(async () => {
+    ({ app, ipAddress, origin } = await makeTestApp());
+  });
+
+  describe("Valid requests", () => {
+    it("Should return 200 given a string input", async () => {
+      const response = await request(app)
+        .post(endpointUrl)
+        .set("X-Forwarded-For", ipAddress)
+        .set("Origin", origin)
+        .send({
+          model: "mongodb-chat-latest",
+          stream: true,
+          input: "What is MongoDB?",
+        });
+
+      expect(response.statusCode).toBe(200);
+      expect(response.body).toEqual({ status: "ok" });
+    });
+
+    it("Should return 200 given a message array input", async () => {
+      const response = await request(app)
+        .post(endpointUrl)
+        .set("X-Forwarded-For", ipAddress)
+        .set("Origin", origin)
+        .send({
+          model: "mongodb-chat-latest",
+          stream: true,
+          input: [
+            { role: "system", content: "You are a helpful assistant." },
+            { role: "user", content: "What is MongoDB?" },
+            { role: "assistant", content: "MongoDB is a document database." },
+            { role: "user", content: "What is a document database?" },
+          ],
+        });
+
+      expect(response.statusCode).toBe(200);
+      expect(response.body).toEqual({ status: "ok" });
```

Comment:
```suggestion
      expect(response.statusCode).toBe(200);
```

nit: since the response object will change considerably, i'd recommend ommitting the response.body check. i think just checking the status as you do in the above line is sufficient for these tests. comment applies to all other tests

Reply:
That makes sense! I was thinking about using these tests to expand on more than just validation the schema, at which point these tests would fail (for good reason). But I'm down with that too, I'll get this updated!